NSS-Day-Cohort-49/giffygram-ermagerds
# updated main to fetch data before rendering. login screen functions a…

Steps to review

#### Changes Made
1. modified main.js to fetch before rendering
2. Added event listener in main to re-render html  (when login button is clicked)
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Logging in with a valid user id should store the user info in locale storage and re-render HTML showing giffygram home page
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
